test/specs/rules/require-sorted-object-keys.js

```diff
@@ -0,0 +1,49 @@
+var Checker = require('../../../lib/checker');
+var assert = require('assert');
+
+describe('rules/require-sorted-object-keys', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+    });
+
+    it('should alias option value true to asc', function() {
+        checker.configure({ requireSortedObjectKeys: true });
+
+        assert(checker.getConfiguration().getConfiguredRule('requireSortedObjectKeys').sort === 'asc');
```

Comment:
This reaches into a rule's implementation. Would prefer to assert the implied sort order with a test like the other values (i.e., testing that it reports on descending keys but not on ascending)